apps/src/gamelab/AnimationTab/index.jsx

```diff
@@ -31,28 +34,48 @@ var staticStyles = {
 /**
  * Root of the animation editor interface mode for GameLab
  */
-var AnimationTab = function () {
+var AnimationTab = function (props) {
   return (
-    <ResizablePanes style={staticStyles.root}>
-      <div id='sequences-column' style={staticStyles.sequencesColumn}>
-        <GameLabVisualizationHeader />
-        <AnimationSequenceList />
-      </div>
-      <div id='frames-column' style={staticStyles.framesColumn}>
-        <div className="purple-header workspace-header">
-          <span>Frames</span>
+    <div>
+      <ResizablePanes style={staticStyles.root}>
+        <div id='sequences-column' style={staticStyles.sequencesColumn}>
+          <GameLabVisualizationHeader />
+          <AnimationSequenceList />
         </div>
-        <AnimationFrameList />
-      </div>
-      <div id='editor-column'>
-        <div className="purple-header workspace-header">
-          <span>Workspace</span>
+        <div id='frames-column' style={staticStyles.framesColumn}>
+          <div className="purple-header workspace-header">
+            <span>Frames</span>
+          </div>
+          <AnimationFrameList />
         </div>
-        <div style={staticStyles.editorRegion}>
-          TODO: Piskel editor goes here!
+        <div id='editor-column'>
+          <div className="purple-header workspace-header">
+            <span>Workspace</span>
+          </div>
+          <div style={staticStyles.editorRegion}>
+            TODO: Piskel editor goes here!
+          </div>
         </div>
-      </div>
-    </ResizablePanes>
+      </ResizablePanes>
+      {props.isAnimationPickerShowing &&
+          <AnimationPicker handleClose={props.hideAnimationPicker} />}
```

Comment:
This diff reads better ignoring whitespace.  I've introduced a new wrapper div so that `AnimationPicker` can be a sibling to `ResizablePanes` here, which indents everything two spaces.